integration/src/main/java/scripts/testscripts/DatasetPermissionOperations.java

```diff
@@ -172,7 +172,8 @@ private void testDiscovererPermissionsForSnapshotDataset() throws Exception {
     // TODO (DR-2833): Uncomment once TDR makes retrieve snapshot work for discoverers.
```

Comment:
To avoid checking in commented out code, you could instead move this code to the JIRA ticket.

Reply:
I'll make a corresponding ticket for our board.